enhancements/declarative-index-config.md

```diff
@@ -0,0 +1,483 @@
+                    {
+                        "group": "etcd.database.coreos.com",
+                        "version": "v1alpha1",
+                        "kind": "etcd"
+                    }
+                ],
+                "requiredAPIs": []
+            },
+            {
+                "path": "quay.io/somethingelse/etcdv0.0.2",
+                "version": "v0.0.2",
+                "channel": "alpha",
+                "upgradeMode": "semver",
+                "ownedAPIs": [
+                    {
+                        "group": "etcd.database.coreos.com",
+                        "version": "v1alpha1",
+                        "kind": "etcd"
+                    }
+                ],
+                "requiredAPIs": []
+            }
+        ]
+    }
+```
+
+#### Story 3
+
+As an index author(with push permission to the namespace my index is hosted in), I can edit the structure of my index (remove bundles from index, change channel bundle is in), using only the json representation of the index/packages inside the index. 
+
+```bash=
+$ cd community-operators 
+$ //TODO come up with sed command to edit "channel":"alpha" to "channel":"beta" for one of the bundles in etcd.json 
+$ cd ..
+$ opm index update community-operators --tag=docker.io/my-namespace/community-operators:latest
+$ rm -rf community-operators
+$ opm index represent packages --index=docker.io/my-namespace/community-operators --output=json
+$ cat community-operators/etcd.json
+"etcd": {
+        "name": "etcd",
+        "defaultChannel": "alpha",
+        "icon": "asdfasdfoipuasdfvasdfqer",
+        "bundles": [
+            {
+                "path": "quay.io/random/etcdv0.0.1",
+                "version": "v0.0.1",
+                "channel": "alpha",
+                "upgradeMode": "semver",
+                "ownedAPIs": [
+                    {
+                        "group": "etcd.database.coreos.com",
+                        "version": "v1alpha1",
+                        "kind": "etcd"
+                    }
+                ],
+                "requiredAPIs": []
+            },
+            {
+                "path": "quay.io/somethingelse/etcdv0.0.2",
+                "version": "v0.0.2",
+                "channel": "beta",
+                "upgradeMode": "semver",
+                "ownedAPIs": [
+                    {
+                        "group": "etcd.database.coreos.com",
+                        "version": "v1alpha1",
+                        "kind": "etcd"
+                    }
+                ],
+                "requiredAPIs": []
+            }
+        ]
+    }
+
+```
+
+Similarly, removing a `.json` from `community-operators` and pushing that update upto the container image removes the bundle etc. 
+
+#### Story 4
+
+As a user with pull permission from the namespace an index is hosted in/as a component of operator-lifecycle-manager, I can validate the structure of the content of an index using the json representation.
```

Comment:
what does "validate" mean in this context?  syntactical validation, or something deeper that actually does lint-like checking for things like stranded bundles versions?